### DISCUSSION TITLE
about the authors ' photos

### DISCUSSION THREAD

**Discussion by mrSaT13** [navidrome/navidrome#3090] [Ideas]
I would like to say a huge thank you for your hard work.but tell me if it is possible to add that the artist's photo is loaded by itself.  or the ability to upload manually. or generate an image created from album cards.

**Comment by deluan** [comment]
Currently it is not possible to upload an image using Navidrome's UI, but you can have images of the artists in your library and ND can pick them up. See https://www.navidrome.org/docs/usage/artwork/#artists
